views/js/pciCreator/dev/delor/pciCreator.js

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright DEPP © 2017 - Ministère de l'éducation nationale  
+*/
+
+define([
+    'lodash',
+    'delor/creator/widget/Widget',
+    'tpl!delor/creator/tpl/markup'
+], function(_, Widget, markupTpl){
+
+    "use strict"; 
+
+    var _typeIdentifier = 'delor';
+
+    var delorCreator = {
+        /**
+         * (required) Get the typeIdentifier of the custom interaction
+         * 
+         * @returns {String}
+         */
+        getTypeIdentifier : function(){
+            return _typeIdentifier;
+        },
+        /**
+         * (required) Get the widget prototype
+         * Used in the renderer
+         * 
+         * @returns {Object} Widget
+         */
+        getWidget : function(){
+            return Widget;
+        },
+        /**
+         * (optional) Get the default properties values of the pci.
+         * Used on new pci instance creation
+         * 
+         * @returns {Object}
+         */
+        getDefaultProperties : function(pci){
+            
+        },
+        /**
+         * (optional) Callback to execute on the 
+         * Used on new pci instance creation
+         * 
+         * @returns {Object}
+         */
+        afterCreate : function(pci){
+            $("div .delor").each(function(index, el) {
```

Comment:
ajouter une mention todo ici: 
`//TODO - fix me when more than one instance of such PCIs is implemented`